UPDATING.md

```diff
@@ -53,8 +53,10 @@ Migrated are:
 | airflow.hooks.S3_hook.S3Hook                                 | airflow.providers.aws.hooks.s3.S3Hook                    |
```

Comment:
Those need to be updated as well.

Reply:
the s3hook isn't part of this PR/ticket, that was #6443 - i don't mind also moving all of the providers/aws/* into providers/amazon/aws/*  as part of this PR, if wanted...